src/components/CommentUpload.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { API, Auth } from 'aws-amplify';
+import {
+  Button, Col, Container, Form, Row,
+} from 'react-bootstrap';
+
+const propTypes = {
+  videoID: PropTypes.number.isRequired,
+};
+
+class CommentUpload extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      content: '',
+      timestamp: '',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.formIsInvalid = this.formIsInvalid.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      [event.target.id]: event.target.value,
+    });
+  }
+
+  defaultTimestamp() {
+    const { timestamp } = this.state;
+    if (timestamp === '') {
+      return -1;
+    }
+    return timestamp;
+  }
+
+  async handleSubmit(event) {
+    const { content } = this.state;
+    const { videoID } = this.props;
+
+    event.preventDefault();
+
+    try {
+      const user = await Auth.currentAuthenticatedUser();
+
+      await API.post('videocloud', '/comments', {
+        body: {
+          content,
+          video: videoID,
+          timestamp: this.defaultTimestamp(),
+          created_by: user.attributes['custom:id'],
+        },
+      });
+      alert('Comment submitted');
+    } catch (error) {
+      alert(error);
+    }
+  }
+
+  formIsInvalid() {
+    const { content } = this.state;
+    return content.trim().length <= 0;
+  }
+
+  render() {
+    const { content, timestamp } = this.state;
+
+    return (
+      <Container className="CommentUpload">
+        <Form onSubmit={this.handleSubmit}>
+          <Row>
+            <Col md={9}>
+              <Form.Group controlId="content">
+                <Form.Control
+                  onChange={this.handleChange}
+                  value={content}
+                  placeholder="Write a comment..."
+                />
+              </Form.Group>
+            </Col>
+            <Col md={3}>
+              <Form.Group controlId="timestamp">
+                <Form.Control
+                  type="number"
+                  onChange={this.handleChange}
+                  value={timestamp}
```

Comment:
This is a nice idea and can stay for now, but in the future the timestamp should be automatically placed.

Reply:
Yes, it's more so that you can do some testing with creating comments to the DB with timestamps.

Basic comments that are not posted as 'timestamp-comments' are for now stored in the DB with value -1 on the timestamp column. Will this work well with the future comment implementations you think?